Sources/ArgumentParserToolInfo/ToolInfo.swift

```diff
@@ -51,6 +51,8 @@ public struct CommandInfoV0: Codable, Hashable {
   public var abstract: String?
   /// Extended description of the command's functionality.
   public var discussion: String?
+  /// Command should appear in help displays.
+  public var shouldDisplay: Bool
```

Comment:
can you add ` = true` as a fall back for older json generated before this option was introduced?

Reply:
Updated now - apologies for the delay, didn't have my laptop with me for the past week.